### ISSUE TITLE
CI setup attempts to fetch from travis-ci.rstudio.org fails under bionic

### ISSUE THREAD

**Issue by eddelbuettel** [Rdatatable/data.table#4672]
PR #4671 filed yesterday eve failed to complete CI; I assumed a temporary glitch.  Looking now, some 12+ hours later I get nothing but 403s for the URLs by 

```sh
 curl -fLo /tmp/R-4.0.0-$(lsb_release -cs).xz https://travis-ci.rstudio.org/R-4.0.0-$(lsb_release -cs).xz
```

as well as any reasonable variants (modifying 4.0.0 to the current 4.0.2, and trying different release names).  

I looks like the 4.0.0-xenial combination it the only one that works, but the .travis.yaml script is using bionic.  The overall code is a little involved (hi @jangorecki ;-) ) but maybe someone could check if it is all still as planned?

**Comment by jangorecki** [comment]
I think macosx build is usually disabled and enabled only before release to CRAN. It is disabled in current master so just merging your PR to current master should "resolve" your problem :)
Problem with Travis is that you can never reproduce it in-house which discourage from debugging.  Thanks for figuring out the root cause, it will definitely help to fix the issue, if it will not go away itself till then :)

**Comment by jangorecki** [comment]
Issue has been resolved somewhere upstream in travis osx R configuration.
build on travis osx is now passing fine (having merged #4743): https://travis-ci.org/github/Rdatatable/data.table/jobs/735041617
